apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c

```diff
@@ -218,6 +219,11 @@ static void utc_wifi_manager_get_mode_p(void)
 
 	TC_ASSERT_EQ("wifi_manager_get_mode_p", ret, WIFI_MANAGER_SUCCESS);
 	TC_ASSERT_EQ("wifi_manager_get_mode_p", info.mode, SOFTAP_MODE);
+	printf("\nMAC Address: ");
+	for (i = 0; i < 5; i++) {
+		printf("%x:", info.mac_address[i]);
```

Comment:
You define the info variable as wifi_manager_info_s.
That structure doesn't have the mac_address element.
Maybe it should be wifi_utils_info.

Reply:
I add mac_address into wifi_manager_info_s.